pgjdbc/src/test/java/org/postgresql/test/jdbc2/DriverTest.java

```diff
@@ -125,25 +128,30 @@ public void testAcceptsURL() throws Exception {
     assertFalse(drv.acceptsURL("jdbc:postgresql://localhost:-2/test"));
 
     // failover urls
-    verifyUrl(drv, "jdbc:postgresql://localhost,127.0.0.1:5432/test", "localhost,127.0.0.1",
-        "5432,5432", "test");
-    verifyUrl(drv, "jdbc:postgresql://localhost:5433,127.0.0.1:5432/test", "localhost,127.0.0.1",
-        "5433,5432", "test");
-    verifyUrl(drv, "jdbc:postgresql://[::1],[::1]:5432/db", "[::1],[::1]", "5432,5432", "db");
-    verifyUrl(drv, "jdbc:postgresql://[::1]:5740,127.0.0.1:5432/db", "[::1],127.0.0.1", "5740,5432",
-        "db");
+    verifyUrl(drv, "jdbc:postgresql://localhost,127.0.0.1:5432/test", "localhost,127.0.0.1", "5432,5432", "test", "osUser", null);
+    verifyUrl(drv, "jdbc:postgresql://localhost:5433,127.0.0.1:5432/test", "localhost,127.0.0.1", "5433,5432", "test", "osUser", null);
+    verifyUrl(drv, "jdbc:postgresql://[::1],[::1]:5432/db", "[::1],[::1]", "5432,5432", "db", "osUser", null);
+    verifyUrl(drv, "jdbc:postgresql://[::1]:5740,127.0.0.1:5432/db", "[::1],127.0.0.1", "5740,5432", "db", "osUser", null);
   }
 
-  private void verifyUrl(Driver drv, String url, String hosts, String ports, String dbName)
+  private void verifyUrl(Driver drv, String url, String hosts, String ports, String dbName, String user, @Nullable String pass)
```

Comment:
How about adding an overload for the old private `verifyUrl(...)` to call the new one?

```java
private void verifyUrl(Driver drv, String url, String hosts, String ports, String dbName) {
    verifyUrl(drv, url, hosts, ports, dbName, "osUser", null);
}
```

That way the majority of the above test cases would not duplicate that info and it would better showcase that none of the expected behavior for those tests have changed.

Reply:
duplicate info removed: https://github.com/pgjdbc/pgjdbc/pull/2646/commits/c7413f775fbc901ac952c70880639ca6db9013db